alonzo/impl/test/test/Test/Cardano/Ledger/Alonzo/DescribeEra.hs

```diff
@@ -0,0 +1,22 @@
+module Test.Cardano.Ledger.Alonzo.DescribeEra (describeTest) where
```

Comment:
Likewise move to `cardano-ledger-test`, remove reference to Alonzo in module name.